lib/fizzy/execute.cpp

```diff
@@ -229,6 +229,31 @@ execution_result execute(Instance& instance, FuncIdx function, std::vector<uint6
             break;
         case Instr::end:
             goto end;
+        case Instr::call:
+        {
+            const auto idx = read<uint32_t>(immediates);
+            assert(idx < instance.module.typesec.size());
+            auto inputs = instance.module.typesec[idx].inputs.size();
```

Comment:
I don't think this is correct, immediate of `call` is the function index, not the type index... so we need to parse function section for this